components/combobox/stories/template.js

```diff
@@ -0,0 +1,135 @@
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+
+import { Template as Menu } from '@spectrum-css/menu/stories/template.js';
+import { Template as TextField } from '@spectrum-css/textfield/stories/template.js';
+import { Template as Popover } from '@spectrum-css/popover/stories/template.js';
+import { Template as PickerButton } from '@spectrum-css/pickerbutton/stories/template.js';
+
+import { useArgs, useGlobals } from "@storybook/client-api";
+
+import "../index.css";
+
+export const Template = ({
+  rootClass = "spectrum-Combobox",
+  id,
+  content,
+  customClasses = [],
+  size = "m",
+  isOpen = true,
+  isInvalid = false,
+  isValid = false,
+  isQuiet = false,
+  isDisabled = false,
+  isFocused = false,
+  isKeyboardFocused = false,
+  isLoading = false,
+  isRequired = false,
+  readOnly = false,
+  ...globals
+}) => {
+  const [_, updateArgs] = useArgs();
+  const [{lang}] = useGlobals();
+
+  const { express } = globals;
+
+  try {
+    if (!express) import(/* webpackPrefetch: true */ "../themes/spectrum.css");
+    else import(/* webpackPrefetch: true */ "../themes/express.css");
+  } catch (e) {
+    console.warn(e);
+  }
+
+  return html`
+    <div
+      class=${classMap({
+        [rootClass]: true,
+        [`${rootClass}--size${size?.toUpperCase()}`]: typeof size !== "undefined",
+        'is-open': !isDisabled && isOpen,
+        [`${rootClass}--quiet`]: isQuiet,
+        'is-invalid': !isDisabled && isInvalid,
+        'is-valid': !isDisabled && isValid,
+        'is-focused': !isDisabled && isFocused,
+        'is-keyboardFocused': !isDisabled && isKeyboardFocused,
+        'is-loading': isLoading,
+        'is-disabled': isDisabled,
+        ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
+      })}
+      id=${ifDefined(id)}
+      aria-disabled=${isDisabled ? "true" : "false"}
+      aria-invalid=${ifDefined(isInvalid && !isDisabled ? "false" : undefined)}
+      aria-readonly=${ifDefined(readOnly ? "true" : "false")}
+      aria-required=${ifDefined(isRequired ? "true" : "false")}
+      aria-haspopup="dialog"
+    >
+      ${[
+        TextField({
+          ...globals,
+          size,
+          isQuiet,
+          isDisabled,
+          isValid,
+          isInvalid,
+          isFocused,
+          isKeyboardFocused,
+          // @todo Add isLoading when TextField story has option to show ProgressCircle like the Combobox examples.
```

Comment:
My TextField changes have been merged into main and I don't believe I added this. @castastrophe should we open a new PR to add this functionality to Textfield or can Josh include it with this one?

Reply:
Hmm is it new functionality specifically made possible by these combobox updates or just something we noticed when we were in here?

Reply:
The design spec for combobox has the progress circle on it, and it was added to the example markup.